src/App.js

```diff
@@ -1,25 +1,16 @@
-import logo from './logo.svg';
 import './App.css';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Let's try to not comment linters as they help us write cleaner code.

Reply:
Hi @Ghiftee 

Please note that I've installed propTypes and react-router-dom and they **are** listed in my package.json file however VS Code still gives me an error ('react-router-dom' should be listed in the project's dependencies. Run 'npm i -S react-router-dom' to add it) - this is why I included the disable comment. Please refer to the screenshot below:

![image](https://user-images.githubusercontent.com/11477535/215050360-badd8fb5-e25d-4b8d-b984-711849b7db81.png)